docs/tac-skills.md

```diff
@@ -35,17 +35,31 @@ aea add skill tac_control
 aea install
 ```
 
-### Update the game parameters
-You can change the game parameters in `tac_controller/skills/tac_control/skill.yaml` under `Parameters`.
+Add the following configs to the aea config:
+``` yaml
+ledger_apis:
+  - ledger_api:
+      addr: https://ropsten.infura.io/v3/f00f7b3ba0e848ddbdc8941c527447fe
+      ledger: ethereum
+      port: 3
+```
 
-or use the following comand:
+Set the default ledger to ethereum:
 ``` bash
-aea config set skills.tac_control.shared_classes.parameters.args.start_time 'Nov 10 2019 10:40AM'
+aea config set set agent.default_ledger ethereum
 ```
 
-You must set the start time to a point in the future `start_time: Nov 10 2019  10:40AM`.
+### Update the game parameters
+You can change the game parameters in `tac_controller/skills/tac_control/skill.yaml` under `Parameters`.
 
+You must set the start time to a point in the future `start_time: 12 11 2019  15:01`.
```

Comment:
what about mentioning this command?
`date "+%d %m %Y %H:%M" -d "$dd 1 min"`

Reply:
In one command, that would be:
```
aea config set skills.tac_control.shared_classes.parameters.args.start_time "$(echo `date "+%d %m %Y %H:%M" -d "$dd 1 min"`)"\n
```

Reply:
Thanks, but I feel like this is potentially more confusing than helpful (if the developer is not familiar with the command it will take them a time to understand it.